cirq/contrib/paulistring/pauli_string_phasor.py

```diff
@@ -83,14 +77,12 @@ def _with_half_turns(self, half_turns: Union[float, value.Symbol]
                          ) -> 'PauliStringPhasor':
         return PauliStringPhasor(self.pauli_string, half_turns=half_turns)
 
-    def extrapolate_effect(self, factor: Union[float, value.Symbol]
-                           ) -> 'PauliStringPhasor':
-        return self._with_half_turns(self.half_turns * factor)  # type: ignore
-
     def __pow__(self, power: Union[float, value.Symbol]) -> 'PauliStringPhasor':
-        if power != 1 and self._is_parameterized_():
+        if isinstance(power, value.Symbol) and self.half_turns != 1:
             return NotImplemented
-        return self.extrapolate_effect(power)
+        if isinstance(self.half_turns, value.Symbol) and power != 1:
+            return NotImplemented
+        return self._with_half_turns(self.half_turns * power)
```

Comment:
probably I'm missing something I don't see how this works if power is a Symbol and half_turns == 1.  This would then give float * Symbol which we haven't defined yet?